araproc/analysis/standard_reco.py

```diff
@@ -417,7 +431,7 @@ def lookup_arrival_time(self,
                                                                                        )
         return arrival_time
 
-    def __get_correlation_function(self, ch1, ch2, wavepacket):
+    def __get_correlation_function(self, ch1, ch2, wavepacket, applyHilbert):
```

Comment:
Since we're planning on using correlation functions outside of this class, should we drop the double underscore in the naming of this function?

Reply:
Are we planning to? I thought the plan was to move csw into standard reco for the final commit?

Reply:
Oh gosh my bad are we? Recinded

Reply:
No worries!